experiments/IB_collab_mode.py

```diff
@@ -47,8 +61,26 @@ def create_15km_scheme():
 
 class IBCollabMode(ExperimentPrototype):
 
-    def __init__(self, freq=10800):
+    def __init__(self):
         cpid = 3700  # allocated by Marci Detwiller 20200609
+
+        ib_file = IB_FILE.format(scf.opts.site_id)
+        with open(ib_file) as f:
+            lines = f.readlines()
+
+        time = datetime.datetime.utcnow()
+
+        for line in lines:
+            ll = line.split()
+            timestamp = int(ll[0])
+
+            dt = datetime.datetime.utcfromtimestamp(timestamp)
+
+            if dt < time:
+                continue
```

Comment:
can we add a default freq in here? I can see this failing to define freq if we forget to update the ib_file

Reply:
could also spit a warning message as well that the default will be used

Reply:
We could. I have the same concern that I did with epop that if we use a default, even with a warning its really easy to go a long time without realizing we are doing things wrong. At least if it hard fails we know immediately.

Reply:
ya but problem with hard fail is the radar won't be running

Reply:
some data is better than none I think.